### ISSUE TITLE
Style focused input borders consistently

### ISSUE THREAD

**Issue by barklund** [GoogleForCreators/web-stories-wp#3969]
## Bug Description

Three inputs need correct `whiteout` border:

1. Search input in media panel
1. Color hex input (in panel, not color picker)
1. Color opacity input (in panel, not color picker)

Issue comes from [2048 (comment)](https://github.com/google/web-stories-wp/issues/2084#issuecomment-673879618).

## Expected Behaviour

* Border should be `whiteout`, not `white` nor none.

## Steps to Reproduce

1. Focus any of the above mentioned input fields
1. See wrong border color

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

#### Media search

1. Focus media search input in the media panel
1. Observe a faint white outline on the input field

#### Color hex

1. Click the empty background element
1. Focus background color hex input field in the design panel
1. Observe a faint white outline on the input field (and not a fully white outline as before)

#### Color opacity

1. Add a text element to a page
1. Focus text color opacity input field in the design panel
1. Observe a faint white outline on the input field (and not a fully white outline as before)

**Comment by csossi** [comment]
Verified in QA
